diff --git a/node_modules/sanity/lib/index.js b/node_modules/sanity/lib/index.js
index 814ebb9..af84896 100644
--- a/node_modules/sanity/lib/index.js
+++ b/node_modules/sanity/lib/index.js
@@ -22450,12 +22450,13 @@ const Item = forwardRef(function(props2, ref) {
-  const $ = c(10);
-  let rest, sortable;
+  const $ = c(11);
+  let rest, sortable, disableTransition;
   $[0] !== props2 ? ({
     sortable,
+    disableTransition,
     ...rest
-  } = props2, $[0] = props2, $[1] = rest, $[2] = sortable) : (rest = $[1], sortable = $[2]);
+  } = props2, $[0] = props2, $[1] = rest, $[2] = sortable, $[3] = disableTransition) : (rest = $[1], sortable = $[2], disableTransition = $[3]);
   let t0;
-  $[3] !== ref || $[4] !== rest || $[5] !== sortable ? (t0 = sortable ? /* @__PURE__ */ jsx(SortableListItem, { ref, ...rest }) : /* @__PURE__ */ jsx(ListItem$2, { ref, ...rest }), $[3] = ref, $[4] = rest, $[5] = sortable, $[6] = t0) : t0 = $[6];
+  $[4] !== ref || $[5] !== rest || $[6] !== sortable ? (t0 = sortable ? /* @__PURE__ */ jsx(SortableListItem, { ref, disableTransition, ...rest }) : /* @__PURE__ */ jsx(ListItem$2, { ref, ...rest }), $[4] = ref, $[5] = rest, $[6] = sortable, $[7] = t0) : t0 = $[7];
   let t1;
-  return $[7] !== props2.id || $[8] !== t0 ? (t1 = /* @__PURE__ */ jsx(SortableItemIdContext.Provider, { value: props2.id, children: t0 }), $[7] = props2.id, $[8] = t0, $[9] = t1) : t1 = $[9], t1;
+  return $[8] !== props2.id || $[9] !== t0 ? (t1 = /* @__PURE__ */ jsx(SortableItemIdContext.Provider, { value: props2.id, children: t0 }), $[8] = props2.id, $[9] = t0, $[10] = t1) : t1 = $[10], t1;
 });
 
 
